frontend/src/types/analytics.ts

```diff
@@ -216,6 +216,18 @@ export type Events = {
     /** The permalink to the creator's profile */
     url: string
   }
+  /**
+   * Description: The user visits a source's link in the single result UI
+   * Questions:
+   *   - Are source links clicked much? Does Openverse increase visibility
+   *     of included sources?
+   */
+  VISIT_SOURCE_LINK: {
+    /** The unique ID of the media */
+    id: string
+    /** The permalink to the source's homepage */
+    url: string
```

Comment:
How does url get used for filtering? For creator links, it will be a unique URL. Does it need to be the creator link? Could it just be the source slug? That would be easier to filter on most of the time.